src/classes/mp/views/MPRoomViewFFR.as

```diff
@@ -704,7 +704,7 @@ internal class UserPanel extends Sprite
     private var view:MPRoomViewFFR;
     private var room:MPRoomFFR;
 
-    private var name:Text;
+    private var _name:Text;
```

Comment:
That compiler doesn't support `name` as a var identifier? Haha

Reply:
nah, it's specifically upset that `name` already exists as a property on its superclass (Sprite -> DisplayObject looks like?)

Reply:
Ahhh okok. Well may I suggest using something like `panelName` instead? Using a leading underscore for this variable only might be misleading. Nitpick.

Reply:
mm it is a bit inconsistent... [pushed](https://github.com/flashflashrevolution/rCubed/compare/99251ad728b0d513b9f933fd4c793234f431e2c7..6af39c22cb2c5fe5978cc6c036f0d56017a08ac9)